src/checkers/inference/InferenceLauncher.java

```diff
@@ -128,6 +128,7 @@ public void typecheck(String [] javaFiles) {
 
         final CheckerMain checkerMain = new CheckerMain(InferenceOptions.checkerJar, options);
         checkerMain.addToRuntimeClasspath(getInferenceRuntimeJars());
+        checkerMain.addToClasspath(getInferenceRuntimeJars());
```

Comment:
The naming seems a bit strange now. Do we actually need the runtime jars on both paths?
Isn't it enough to just have `checker-qual.jar` on the compilation classpath?

Reply:
There are qualifiers that are not in checker-qual.jar such as ostrusted.

Reply:
We should add a `checker-inference-qual.jar` as the classpath dependency, instead of adding all runtime jars.
Can you add this as a comment here or open an issue?